src/__test__/calculate.test.js

```diff
@@ -0,0 +1,27 @@
+import calculate from '../logic/calculate';
+
+describe('calculate', () => {
+  test('clears the display when AC button is pressed', () => {
+    const data = {
+      total: '0',
+      next: '',
+      operation: '',
+    };
+
+    const newData = calculate(data, 'AC');
+    expect(newData).toEqual(data);
+  });
+  test('display the multiplication operation when clicked', () => {
+    const myCalculate = { total: 15, operation: 'x', next: '' };
+    const data = calculate(myCalculate, 'x');
+    expect(data.operation).toEqual('x');
+  });
+  test('should calculate output when "=" pressed', () => {
+    const data = { total: 7, next: 5, operation: 'x' };
+    const newData = calculate(data, '=');
+
+    expect(newData.total).toEqual('35');
+    expect(newData.next).toEqual('');
+    expect(newData.operation).toEqual('=');
+  });
+});
```

Comment:
- [ ] Kindly add negative and positive tests for all calculations that may occur in this module, I only see a positive test for multiplication.